test/unit/view/drag-handle.spec.js

```diff
@@ -820,9 +958,39 @@ describe('drag handle', () => {
         })).toBe(true);
       });
 
-      it('should not do anything if there is nothing dragging', () => {
+      it('should block a click after a cancel', () => {
```

Comment:
block -> prevent?

Reply:
yep. i have also updated the other places that used the blocked language